tools/clang/tools/dxrfallbackcompiler/dxcvalidator.cpp

```diff
@@ -64,7 +64,7 @@ HRESULT STDMETHODCALLTYPE DxcValidator::Validate(
     IDxcOperationResult *
         *ppResult // Validation output status, buffer, and errors
 ) {
-  return hlsl::validate(pShader, Flags, ppResult);
+  return hlsl::validate(pShader, Flags, true, ppResult);
```

Comment:
This should not be true, and I don't think we should have this implementation of IDxcValidator in dxrfallbackcompiler in the first place.